src/winetricks

```diff
@@ -14953,6 +14953,45 @@ load_openwatcom()
 
 #----------------------------------------------------------------
 
+w_metadata powershell apps \
+    title="Windows Powershell" \
+    publisher="Microsoft" \
+    year="2019" \
+    media="download" \
+    file1="PowerShell-6.1.6-win-x86.zip" \
+    installed_file1="$W_SYSTEM32_DLLS_WIN/WindowsPowerShell/v1.0/powershell.exe"
+
+load_powershell()
+{
+    w_package_unsupported_win64
```

Comment:
Can be removed, as you said (proper 64-bit support can be a followup PR).

Reply:
I wasn't planning on adding win64 support